gateway/config/development.lua

```diff
@@ -1,7 +1,27 @@
+local cjson = require('cjson')
+
+local configuration = cjson.encode(cjson.decode([[{
+  "services": [
+    {
+      "proxy": {
+        "hosts": [
+          "localhost",
+          "127.0.0.1"
+        ],
+        "policy_chain": [
+          { "name": "apicast.policy.echo" }
+        ]
+      }
+    }
+  ]
+}
+]]))
+
 return {
     worker_processes = '1',
     master_process = 'off',
-    lua_code_cache = 'off',
+    lua_code_cache = 'on',
     configuration_loader = 'lazy',
     configuration_cache = 0,
+    configuration = string.format([[data:application/json,%s]],ngx.escape_uri(configuration)),
```

Comment:
Can you add a link here to https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs ?

Reply:
Done.